src/sentry/static/sentry/app/components/events/interfaces/breadcrumbs/httpRenderer.jsx

```diff
@@ -10,11 +12,17 @@ class HttpRenderer extends React.Component {
   };
 
   renderUrl = url => {
-    return typeof url === 'string' && url.match(/^https?:\/\//) ? (
-      <a href={url}>{url}</a>
-    ) : (
-      <em>{url}</em>
-    );
+    if (typeof url === 'string') {
+      return url.match(/^https?:\/\//) ? <a href={url}>{url}</a> : <em>{url}</em>;
+    }
+
+    Sentry.withScope(scope => {
+      scope.setExtra('url', url);
+      scope.setLevel('info');
+      Sentry.captureException(new Error('Invalid breadcrumb URL'));
```

Comment:
Not sure we need this tbh

Reply:
Probably wouldn't log an error here, since sending a valid URL here is merely a "convention" iirc. At least, there's no validation for this.